### PR TITLE
Redesign "Welcome" toast

### PR DESCRIPTION
**Pull request by hotzenklotz**
PR redesigns the "Welcome" toast for new, anonymous users to be in line with the new branding.

image

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Set dev environment to Wkorg
- Share a dataset with yourself
- Open DS in incognito mode 
- Check out the redesigned welcome toast

### Issues:
- contributes to #6739 

------
(Please delete unneeded items, merge only when none are left open)
- [x] Updated [changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [ ] Updated [migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable
- [ ] Updated [documentation](../blob/master/docs) if applicable
- [ ] Adapted [wk-libs python client](https://github.com/scalableminds/webknossos-libs/tree/master/webknossos/webknossos/client) if relevant API parts change
- [ ] Removed dev-only changes like prints and application.conf edits
- [x] Considered [common edge cases](../blob/master/.github/common_edge_cases.md)
- [ ] Needs datastore update after deployment

### CODE REVIEW

File: frontend/stylesheets/main.less [hunk 1]
```diff
@@ -418,14 +411,37 @@ i.without-icon-margin,
...
+      display: block;
+      position: absolute;
+      top: 0px;
+      right: 0px;
+      content: " ";
+      transform: translate(15%, -104%);
+      background-image: url(/assets/images/drawings/waving-person.svg);
+      background-repeat: no-repeat;
+      background-size: cover;
+    }
```
Comment (philippotto) [inline frontend/stylesheets/main.less:437]: the indention seems off here

**Review by philippotto — COMMENTED** [review]
cool stuff :+1: will test in a bit.

File: frontend/stylesheets/main.less (hunk 1 above)
Comment (hotzenklotz) [reply frontend/stylesheets/main.less]: Fixed

File: public/images/logo-icon-only.svg
[the remark this answers is not in the document]
Comment (hotzenklotz) [reply public/images/logo-icon-only.svg:1]: No, I only meant to rename it to make it clearer. Fixed in PR feedback commit.

**Review by philippotto — APPROVED** [review]
Vorzüglich :+1:

**Pull request merged**